cobra/test/__init__.py

```diff
@@ -50,7 +53,7 @@ def test_all(args=None):
 
         return pytest.main(
             ['--pyargs', 'cobra', '--benchmark-skip', '-v', '-rs'] + args
-        ) != 0
+        )
```

Comment:
Ah, probably easiest to revert that. It would return the code that `pytest.main` gives. So `0` if everything passes as expected and `1` otherwise. Would `test_all` evaluating to `True` be more logical when all tests pass?